### DISCUSSION TITLE
studio_code field for the scene filename parser?

### DISCUSSION THREAD

**Discussion by brokedarius** [stashapp/stash#4459] [Ideas]
Hi folks, I was wondering why there was no studio code field for the scene filename parser? I would be nice and easy to implement as well as very useful since it's often on files. Thanks again for letting me know.

**Comment by DogmaDragon** [comment]
Probably a simple oversight. Not everyone is familiar with the whole codebase or use all the features. It's also the reason studio code was added only to scenes at first (https://github.com/stashapp/stash/pull/3051) and only later to images (https://github.com/stashapp/stash/pull/4217) and galleries (https://github.com/stashapp/stash/pull/4195).

**Comment by mr-szgz** [comment]
for my unmatched OF content 🙏

this would be very nice - would let me RoU these files w/o loosing the little metadata packs can provide
